### DISCUSSION TITLE
Image & Avatar Component

### DISCUSSION THREAD

**Discussion by heysagnik** [heroui-inc/heroui#269] [Feedback]
1. While researching I found a awkward result in Avatar component:
   `<Avatar src="" css={{ size: "$20" }} />`
 ![image](https://user-images.githubusercontent.com/70798495/153444523-ef7c7b63-b785-4fa2-8cc5-d26ccbf81969.png)
Please avoid the background color!
 But according the documentation it should show :
![image](https://user-images.githubusercontent.com/70798495/153444948-8000c3e8-75f8-4ed4-84ae-df966e5dbbfe.png)

2. `<Image src="/avatars/avatar-4.png"/>` what ever is the value of src it is showing a blank white placeholder !!

Please check is this really a bug or I made a mistake in code !

**Comment by jrgarciadev** [comment]
Thank you for reporting it @devgossips 👍🏻 , I'll covert this in an issue
